SayTheirNames/Source/Controller/Home/CollectionViewDataSource/LocationCollectionViewDataSource.swift

```diff
@@ -39,12 +39,14 @@ final class LocationCollectionViewDataSourceHelper {
 
         self.dataSource =
             LocationCollectionViewDataSource(collectionView: collectionView) { (collectionView, indexPath, location) -> UICollectionViewCell? in
-            let cell: LocationCell = collectionView.dequeueCell(for: indexPath)
-            cell.configure(with: location)
-            cell.accessibilityIdentifier = "locationCell\(indexPath.item)"
-            cell.isAccessibilityElement = true
-            cell.accessibilityLabel = "\(location.name)"
-            return cell
+                let cell: LocationCell = collectionView.dequeueCell(for: indexPath)
+                cell.configure(with: location)
+                cell.accessibilityIdentifier = "locationCell\(indexPath.item)"
+                cell.isAccessibilityElement = true
+                let index = indexPath.row + 1
+                let total = collectionView.numberOfItems(inSection: 0)
```

Comment:
Can we put this in an extension or something? I feel like this would be used in other places as well. 

```
extension UICollectionView {

}
```

Reply:
Whats your thought?

Reply:
Outside of that; looks good.

Reply:
@evilpenguin Thanks for the suggestion, taking a look at how I might implement this.